src/main/java/com/onarandombox/MultiverseCore/configuration/EntryFee.java

```diff
@@ -51,6 +53,9 @@ public double getAmount() {
      */
     @Nullable
     public Material getCurrency() {
+        if (currency == null || currency.equals(DISABLED_MATERIAL)) {
+            return null;
+        }
         return currency;
```

Comment:
Simplify these checks into one.
```suggestion
        return currency == Material.AIR ? null : currency;
```

Reply:
Nice suggestion, it's up to ben though.

Reply:
Yea can change that.